cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -78,7 +78,11 @@ private class ServiceImpl(
                 nc.sendConsent(it, env, consentActionImpl)
             }
             .executeOnRight {
-                dataStorage.saveLocalState(it.localState)
+                dataStorage.run {
+                    saveLocalState(it.localState)
+                    println("savedConsent = true")
```

Comment:
@carmelo-iriti is this `println` desirable?

Reply:
Right , let me remove it